scripts/deploy/aks-install-mayastor.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+readonly PKG_ROOT="$(git rev-parse --show-toplevel)"
+
+kubectl label nodes --selector agentpool=agentpool openebs.io/engine=mayastor --overwrite
```

Comment:
When we label all the agent nodes to be "storage nodes" with the current cluster setup, there not enough CPU left to run the core agent. We either should not mark all the nodes as storage, i.e. openebs.io/engine=mayastor or we could increase the CPU for the nodes.

Reply:
by cluster setup I am referring to the [azure-cluster-up.sh](https://github.com/abhisheksinghbaghel/mayastor/blob/develop/scripts/deploy/azure-cluster-up.sh) script Akash has added to the repo.

Reply:
For now, let's just use a bigger SKU.

Reply:
I faced the same issue that @landreasyan mentioned. The core-agents pod takes up a lot of CPU and on whichever node that is running mayastor pods are not able to run. We should probably run the control plane(core-agents) and data plane on different nodes.

Reply:
In an ideal set-up like AKS, we will run control-plane separately. However, when we start asking for separate nodes for each components, then we are moving away from the support in smaller clusters. 
For now lets use bigger SKU to unblock, but in a longer run we should strive to be able to run all this on a single node given minimum required resources.